### PR TITLE
Updated all server code to new repo

### PR DESCRIPTION
**Pull request by andrew-hu**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -16,44 +16,133 @@ app.get('/', function(req,res){
...
+            //console.log(result[2]);
+
+
+            //byte = "0x" + ascii_to_hexa(byte);
+            //byte = parseInt(byte);
+            //socket.end(eval(data));
+        });
+    });
+
+    server.listen(5001, '192.168.43.12');
```
Comment (jpan127) [inline server/app.js:94]: These 2 fields would be useful to display on the page too.

File: server/app.js [hunk 2]
```diff
@@ -16,44 +16,133 @@ app.get('/', function(req,res){
...
 	socket.on('stream',function(image){
 		socket.broadcast.emit('stream', image);
 	});
+    socket.on('err', function(err){
+            // handle the error safely
+            console.log(err)
+        });
 	socket.on('event',function(data){
-		console.log('A client sent us this SMART message:', data.message);
+        client.connect(5000, "192.168.43.250", function() { //192.168.43.250 port 5000
```
Comment (jpan127) [inline server/app.js:107]: Same with these.

File: server/app.js [hunk 3]
```diff
@@ -63,4 +152,20 @@ io.on('err', function(err){
...
+
+http.on('err', function(err){
+            // handle the error safely
+            console.log(err)
+        });
+
+function ascii_to_hexa(str)
+  {
+    var arr1 = [];
+    for (var n = 0, l = str.length; n < l; n ++)
```
Comment (jpan127) [inline server/app.js]: Why create variable `l` when you can just:
```
for (var n = 0; n < str.length; n++)
```

**Review by jpan127 — APPROVED** [review]

**Comment by jpan127** [comment]
Is this still going to be merged?

**Comment by andrew-hu** [comment]
Not going to merge this old PR, will include all in next one

**Pull request closed without merging**